### ISSUE TITLE
Bootswatch "Materia" theme: dateRangeInput minor UX issue

### ISSUE THREAD

**Issue by MadhulikaTanuboddi** [rstudio/bslib#193]
Run the following..

```
library(shiny)
library(bslib)

ui <- fluidPage(
  theme = bs_theme(bootswatch = "materia"),
  dateRangeInput("date", "Date", start = Sys.Date(), end = Sys.Date() + 7),
)
shinyApp(ui, function(input, output) {})
```

Notice the border line for the second date input starts from "to" (minor issue but noticable)

<img width="495" alt="Materia_theme_dateRangeInput" src="https://user-images.githubusercontent.com/5369655/102402008-04eb2e80-3f99-11eb-8418-9f84e6fec622.png">

**Comment by cpsievert** [comment]
The `selectInput()` styles need to be fixed too (they need an underline). I had made a point to fix in this in #148, but looks like I forgot to when the scope of that PR exploded.

**Comment by MadhulikaTanuboddi** [comment]
<img width="1639" alt="Materia_themese_selectize_Input" src="https://user-images.githubusercontent.com/5369655/103671652-0c667d80-4f30-11eb-8a1b-287b152dfac0.png">

@cpsievert : Fyi, noticed a minor cursor type dot in the selectizeInput box while selecting the options. (seems to be on chrome and safari (Mac), still need to check on windows). Is that expected?
